saleor/dashboard/group/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from . import views
+
+
+urlpatterns = [
+    url(r'^$', views.groups_list, name="group-list"),
```

Comment:
Strings should be consistently enclosed in quotes or double quotes. In Saleor we tend to use single quotes in most cases.

Reply:
old habbits